plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/status/StatusUtilTest.java

```diff
@@ -61,13 +61,15 @@ public void testError_withInstanceAndException() {
 
   @Test
   public void testError_withClassAsObject() {
+    @SuppressWarnings("cast")
     IStatus error = StatusUtil.error((Object) new DeleteAllVisitor().getClass(), "test error msg");
     verifyStatus(error);
   }
 
   @Test
   public void testError_withClassAsObjectAndException() {
     Throwable exception = new Exception();
+    @SuppressWarnings("cast")
     IStatus error =
         StatusUtil.error((Object) new DeleteAllVisitor().getClass(), "test error msg", exception);
```

Comment:
I this these tests are from when we had separate `error(Class,…)` methods.  They can probably just go away.  Or at least use `DeleteAllVisitor.class`?